docs/en/setup/backend/backend-meter.md

```diff
@@ -17,14 +18,29 @@ kafka-fetcher:
     bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
 ```
 
+## Meter collection
+There are plentiful metrics are collected by default, you may find predefined metrics by oal files located in `config/oal/`.
```

Comment:
```suggestion
```
I think we should remove this, due to OAL doesn't relate to MAL at all.

Reply:
https://skywalking.apache.org/docs/main/latest/en/guides/backend-oal-scripts/#official-oal-script This is the doc for OAL